scripts/setup.sh

```
@@ -0,0 +1,20 @@
+fi
+
+if [ -d "manifests" ]; then
+    rm -rf "manifests"
+fi
+
+echo "sozo build && sozo migrate apply"
+sozo build && sozo migrate apply
+
+echo -e "\n✅ Setup finish!"
+
+export world_address=$(cat ./manifests/dev/deployment/manifest.json | jq -r '.world.address')
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)